### DISCUSSION TITLE
Undefined Variable fix

### DISCUSSION THREAD

**Discussion by gladishd** [carbon-design-system/carbon#11537] [Release Feedback]
@import '@carbon/themes/scss/themes';
@import 'carbon-components/scss/globals/scss/styles.scss';

// Use the gray 100 theme
$theme: $theme--g100;
@include theme();

https://github.com/carbon-design-system/carbon-tutorial/issues/3940

**Comment by tay1orjones** [comment]
Thanks for the heads up @gladishd, we're working to update the `@carbon/theme` docs in #11293
